geode-connectors/src/distributedTest/java/org/apache/geode/connectors/jdbc/internal/cli/ListMappingCommandDUnitTest.java

```diff
@@ -74,31 +81,147 @@ public void listsRegionMappingFromClusterConfiguration() throws Exception {
   }
 
   @Test
-  public void listsRegionMappingsFromMember() throws Exception {
-    Properties properties = new Properties();
-    properties.put(DistributionConfig.ENABLE_CLUSTER_CONFIGURATION_NAME, "false");
-
-    locator = startupRule.startLocatorVM(0, properties);
-    server = startupRule.startServerVM(1, locator.getPort());
+  public void reportsNoRegionMappingsFoundForServerGroup() throws Exception {
+    locator = startupRule.startLocatorVM(0);
+    server1 = startupRule.startServerVM(1, TEST_GROUP1, locator.getPort());
     gfsh.connectAndVerify(locator);
     gfsh.executeAndAssertThat("create region --name=" + regionName + " --type=REPLICATE")
         .statusIsSuccess();
 
-    server.invoke(() -> createNRegionMappings(3));
+    String mapping = "create jdbc-mapping --region=" + regionName + " --data-source=connection "
+        + "--table=myTable --pdx-name=myPdxClass";
+    gfsh.executeAndAssertThat(mapping).statusIsSuccess();
+
+    CommandStringBuilder csb = new CommandStringBuilder(LIST_MAPPING + " --groups=" + TEST_GROUP1);
```

Comment:
Can we add one more `list jdbc-mapping` command without `--group` here?

Reply:
✅